test/cfa/ntp_conf_test.rb

```diff
@@ -59,9 +59,11 @@ def ntp_conf(file)
       it "writes a new entry" do
         record = CFA::NtpConf::ServerRecord.new
         record.value = "3.pool.ntp.org"
+        record.raw_options = "iburst"
+        record.comment = nil
```

Comment:
not needed, I just test some options there and in real code this happens sometimes.